src/Comms.cpp

```diff
@@ -2,52 +2,52 @@
 
 namespace Comms {
 
-    // std::map<uint8_t, commFunction> callbackMap;
+    commFunction callbackMap[5];
     // EthernetUDP Udp;
     char packetBuffer[sizeof(Packet)];
 
-    // void registerCallback(uint8_t id, commFunction function) {
-    //     callbackMap.insert(std::pair<int, commFunction>(id, function));
-    // }
+    void registerCallback(uint8_t id, commFunction function) {
+        callbackMap[id] = function;
+    }
 
     /**
      * @brief Checks checksum of packet and tries to call the associated callback function.
      * 
      * @param packet Packet to be processed.
      */
-    // void evokeCallbackFunction(Packet *packet) {
-    //     uint16_t checksum = *(uint16_t *)&packet->checksum;
-    //     if (checksum == computePacketChecksum(packet)) {
-    //         DEBUG("Packet with ID ");
-    //         DEBUG(packet->id);
-    //         DEBUG(" has correct checksum!\n");
-    //         //try to access function, checking for out of range exception
-    //         if(callbackMap.count(packet->id)) {
-    //             callbackMap.at(packet->id)(*packet);
-    //         } else {
-    //             DEBUG("ID ");
-    //             DEBUG(packet->id);
-    //             DEBUG(" does not have a registered callback function.\n");
-    //         }
-    //     } else {
-    //         DEBUG("Packet with ID ");
-    //         DEBUG(packet->id);
-    //         DEBUG(" does not have correct checksum!\n");
-    //     }
-    // }
+    void evokeCallbackFunction(Packet *packet) {
+        uint16_t checksum = *(uint16_t *)&packet->checksum;
+        if (checksum == computePacketChecksum(packet)) {
+            DEBUG("Packet with ID ");
+            DEBUG(packet->id);
+            DEBUG(" has correct checksum!\n");
+            //try to access function, checking for out of range exception
+            if(packet->id < 5) {
+                callbackMap[packet->id](*packet);
+            } else {
+                DEBUG("ID ");
+                DEBUG(packet->id);
+                DEBUG(" does not have a registered callback function.\n");
+            }
+        } else {
+            DEBUG("Packet with ID ");
+            DEBUG(packet->id);
+            DEBUG(" does not have correct checksum!\n");
+        }
+    }
 
     void processWaitingPackets() {
         if(Serial.available()) {
```

Comment:
Serial1?
Why change to Serial1? Will the serial commands/ telemetry be sent to FC/another board instead of directly to a computer?